.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=hca
```

Comment:
It turns out that docker-compose doesn't support underscores or dashes in COMPOSE_PROJECT_NAME, so options are: hcadcp_dss or dcp_dss.  Which would you prefer?